gremlin-python/src/main/python/gremlin_python/process/graph_traversal.py

```diff
@@ -1451,6 +1467,10 @@ def merge_v(cls, *args):
     def min_(cls, *args):
         return cls.graph_traversal(None, None, Bytecode()).min_(*args)
 
+    @classmethod
+    def none(cls, *args):
+        return cls.graph_traversal(None, None, Bytecode()).none(*args)
```

Comment:
If my understanding is correct, `none()` is a terminal step so there is no use to add it in the `__` class here (which is why it hasn't been in the class).

Reply:
Oh right, I forgot that none hasn't been changed to a filtering step yet.  Removing this change.